plugins/generator-1.21.4/neoforge-1.21.4/templates/elementinits/items.java.ftl

```diff
@@ -40,6 +40,8 @@ package ${package}.init;
 <#assign hasBlocks = false>
 <#assign hasDoubleBlocks = false>
 <#assign hasItemsWithProperties = w.getGElementsOfType("item")?filter(e -> e.customProperties?has_content)?size != 0>
+<#assign hasItemsWithLeftHandedProperty = w.getGElementsOfType("item")?filter(e -> e.states

```

Comment:
Move this inside `<#if hasItemsWithProperties>` so this does not evaluate on workspaces where it is not needed